jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.vue

```diff
@@ -8,134 +8,192 @@
     :popout_button="popout_button"
     :disabled_msg="disabled_msg">
 
-    <v-row>
-      <v-select
-        :menu-props="{ left: true }"
-        attach
-        :items="function_items.map(i => i.label)"
-        v-model="function_selected"
-        label="Function"
-        hint="Function for reducing dimensions of spectral cube."
-        persistent-hint
-      ></v-select>
-    </v-row>
-
-    <plugin-subset-select 
-      :items="aperture_items"
-      :selected.sync="aperture_selected"
-      :show_if_single_entry="true"
-      label="Spatial aperture"
-      hint="Select a spatial region to extract its spectrum."
-    />
-
-    <div v-if="aperture_selected !== 'Entire Cube' && dev_cone_support">
-      <v-alert type='warning'>cone support is under active development and hidden from users</v-alert>
-      <v-row>
-        <v-switch
-          v-model="wavelength_dependent"
-          label="Wavelength dependent"
-          hint="Vary aperture linearly with wavelength"
-          persistent-hint>
-        </v-switch>
-      </v-row>
-      <div v-if="wavelength_dependent">
-        <v-row justify="end">
-          <j-tooltip tooltipcontent="Adopt the current slice as the reference wavelength">
-            <plugin-action-button :results_isolated_to_plugin="true" @click="adopt_slice_as_reference">
-              Adopt Current Slice
-            </plugin-action-button>
-          </j-tooltip>
-        </v-row>
+    <div @mouseover="() => active_step='ap'">
+      <j-plugin-section-header :active="active_step==='ap'">Aperture</j-plugin-section-header>
+
+      <plugin-subset-select 
+        :items="aperture_items"
+        :selected.sync="aperture_selected"
+        :show_if_single_entry="true"
+        label="Spatial aperture"
+        hint="Select a spatial region to extract its spectrum."
+      />
+
+      <div v-if="aperture_selected !== 'Entire Cube' && dev_cone_support">
         <v-row>
-          <v-text-field
-            v-model.number="reference_wavelength"
-            type="number"
-            :step="0.1"
-            class="mt-0 pt-0"
-            label="Wavelength"
-            hint="Wavelength at which the aperture matches the selected subset."
-            persistent-hint
-          ></v-text-field>
-        </v-row>
-        <v-row justify="end">
-          <j-tooltip tooltipcontent="Select the slice nearest the reference wavelength">
-            <plugin-action-button :results_isolated_to_plugin="true" @click="goto_reference_wavelength">
-              Slice to Reference Wavelength
-            </plugin-action-button>
-          </j-tooltip>
+          <v-switch
+            v-model="wavelength_dependent"
+            label="Wavelength dependent"
```

Comment:
Should we label this "Conical (wavelength dependent)"?

Reply:
Let's run this by POs and can also update in any of the follow-up works... technically it won't _look_ like a cone in frequency-space or if there are gaps in wavelength... but I agree that the terminology may still be more descriptive at first glance to a user.

Reply:
(@javerbukh - feel free to change this as part of #2679 as well)